x/emissions/types/validations.go

```diff
@@ -0,0 +1,19 @@
+package types
```

Comment:
would have specifically called this file `dec .go` to be in line with others in this dir but nbd

Reply:
the idea is to add more common validations here in the future